### PR TITLE
Refactoring Rasa kick starter document

### PR DESCRIPTION
**Pull request by sshniro**

### CODE REVIEW

File: bots/running-a-rasa-bot/README.md [hunk 1]
```diff
@@ -1,54 +1,33 @@
 # Running a Rasa Bot
 
 Rasa is the leading open-source machine learning toolkit that lets developers expand bots beyond answering simple questions
-with minimal training data. The bots are based on a machine learning model trained on example conversations.
+with minimal training data. At the core Rasa bot has a machine learning model which trained on example conversations.
 
 ## Quickstart
 
-Rasa is developed with Python. For the most part you dont need to know python to design the basic conversational flows in Rasa.
+Rasa is developed with Python, but for the most part you dont need to know python to design the basic conversational flows in Rasa.
```
Comment (bandantonio) [inline bots/running-a-rasa-bot/README.md:8]: no comma needed before `but`

File: bots/running-a-rasa-bot/README.md [hunk 2]
```diff
@@ -1,54 +1,33 @@
 # Running a Rasa Bot
```
Comment (bandantonio) [inline bots/running-a-rasa-bot/README.md:1]: Please check my PR here and refactor all the hedings according to the structure:
https://github.com/RocketChat/docs/pull/1445/files

File: bots/running-a-rasa-bot/README.md [hunk 3]
```diff
@@ -1,54 +1,33 @@
...
-Access <http://localhost:3000/>
-
-Create an `admin` user with the password `admin`.
-
-### 2. Rocket.Chat Bot User Configurations
-
-Clone the quick start guide with the following command: `git clone https://github.com/RocketChat/rasa-kick-starter`
-Create a Rasa bot user in Rocket.Chat. You can either manually login to Rocket.Chat and create a bot user via the
-user management page or can use the following script to create the bot user.
+### Create Rocket.Chat Bot
```
Comment (bandantonio) [inline bots/running-a-rasa-bot/README.md:17]: All the steps related to creating a bot user are already described here: https://rocket.chat/docs/bots/create-and-run-a-bot/#steps

You can safely delete this section

**Review by bandantonio — CHANGES_REQUESTED** [review]

**Comment by bandantonio** [comment]
Working on updates to this PR.

Please, **DO NOT MERGE!**

**Comment by bandantonio** [comment]
Consider closing this PR in favor of #1503

**Comment by sshniro** [comment]
Closing the PR.

**Pull request closed without merging**